app/controllers/organizations_controller.rb

```diff
@@ -67,7 +67,7 @@ def update
         format.html { redirect_to @organization, notice: 'Organization was successfully updated.' }
         format.json { render :show, status: :ok, location: @organization }
       else
-        format.html { render :edit }
+        format.html { redirect_to @organization, alert: 'Organization could not be updated.' }
```

Comment:
feel like we should do something more helpful here, like print the error, but I tried `@organization.errors` and there didn't appear to be anything there. this can occur if you try to remove a contact email, since we require one.